shell/platform/darwin/ios/framework/Source/FlutterRestorationPlugin.mm

```diff
@@ -54,8 +54,7 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
 
 - (void)setRestorationData:(NSData*)data {
   if (data != _restorationData) {
-    [_restorationData release];
-    _restorationData = [data retain];
+    _restorationData = data;
```

Comment:
This doesn't look like it needs to be a copy based on its usage. It's `strong` in the header.

Reply:
I would expect `NSData` to use `copy` for the same reason as `NSString` and the collection types: you never know when someone might pass you the mutable version instead. The framework classes generally use CoW so that `copy` is ~free when not actually needed.

Reply:
Do you think swapping that in the header is a breaking change somehow? 

https://github.com/flutter/engine/blob/11f2cb37c35c7fb28cadaac72bad656ef463d398/shell/platform/darwin/ios/framework/Source/FlutterRestorationPlugin.h#L18

Reply:
In theory it could break us if we have code somewhere relying on passing in `NSMutabledata` and then modifying it after the fact. Hopefully that's not the case; it sounded like you had determined that it wasn't. But I can do a sweep too for safety.

Reply:
I only see it set in one place, and that's from method channel handler arguments, which should definitely not be something that changes after the fact. Switching to `copy` looks safe to me.

Reply:
I had done a sweep and it looked good to me, but for some reason when I asked just now I thought `FlutterRestorationPlugin.h` was in the umbrella header.  That's what I get for doing this at 11:30pm and then paging it all out for a day.  Thanks for double checking me 🙂